starlark/testdata/math.star

```diff
@@ -255,8 +255,8 @@ assert.fails(lambda: math.sin("0"), "got string, want float or int")
 # tan
 assert.eq(math.tan(0.0), 0)
 assert.eq(math.tan(0), 0)
-assert.eq(math.tan(math.pi / 4), 1)
-assert.eq(math.tan(-math.pi / 4), -1)
+assert.true(near(math.tan(math.pi / 4), 1, 0.00000000000001))
```

Comment:
Thanks for the fix. The code change looks good, but it's odd that there are 2 ulps of difference in the result
```
fmt.Printf("%[1]s: %[2]v %.16[2]x\n", runtime.GOARCH, math.Tan(math.Pi / 4))

arm64: 0.9999999999999998 0x1.ffffffffffffe000p-01
amd64: 1 0x1.0000000000000000p+00
```
even though both architectures use the [same software implementation](https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/math/tan.go;l=83;drc=8ac58de1857637f372a00ea16ab5497193b784a6). I wonder if this is due to greater width of Intel's FPU registers.